tests/manage/test_dynamic_pvc_accessmodes_with_reclaim_policies.py

```diff
@@ -0,0 +1,375 @@
+import logging
+import pytest
+
+from ocs_ci.framework.testlib import ManageTest, tier1
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.exceptions import (
+    TimeoutExpiredError, CommandFailed, UnexpectedBehaviour
+)
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources.pvc import PVC
+from ocs_ci.utility import templating
+from tests import helpers
+from tests.fixtures import (
+    create_rbd_secret, create_cephfs_secret, create_ceph_block_pool,
+    create_project
+)
+
+logger = logging.getLogger(__name__)
+
+
+@tier1
+@pytest.mark.usefixtures(
+    create_rbd_secret.__name__,
+    create_cephfs_secret.__name__,
+    create_ceph_block_pool.__name__,
+    create_project.__name__
+)
+class TestRWODynamicPvc(ManageTest):
+    """
+    Automates the following test cases
+    OCS-530 - RBD Based RWO Dynamic PVC creation with Reclaim policy set
+    to Retain
+    OCS-533 - RBD Based RWO Dynamic PVC creation with Reclaim policy set
+    to Delete
+    OCS-525 - CephFS Based RWO Dynamic PVC creation with Reclaim policy set
+    to Retain
+    OCS-526 - CephFS Based RWO Dynamic PVC creation with Reclaim policy set
+    to Delete
+    """
+    @pytest.mark.parametrize(
+        "interface_type,reclaim_policy", [
+            pytest.param(
+                constants.CEPHBLOCKPOOL, constants.RECLAIM_POLICY_RETAIN,
+                marks=pytest.mark.polarion_id("OCS-530")
+            ),
+            pytest.param(
+                constants.CEPHBLOCKPOOL, constants.RECLAIM_POLICY_DELETE,
+                marks=pytest.mark.polarion_id("OCS-533")
+            ),
+            pytest.param(
+                constants.CEPHFILESYSTEM, constants.RECLAIM_POLICY_RETAIN,
+                marks=pytest.mark.polarion_id("OCS-525")
+            ),
+            pytest.param(
+                constants.CEPHFILESYSTEM, constants.RECLAIM_POLICY_DELETE,
+                marks=pytest.mark.polarion_id("OCS-526")
+            )
+        ]
+    )
+    def test_rwo_dynamic_pvc(self, interface_type, reclaim_policy):
+        """
+        Verifies RBD/CephFS Based RWO Dynamic PVC creation with Reclaim policy
+        set to Delete/Retain
+        Steps:
+        1. Create Storage Class with reclaimPolicy: Delete/Retain
+        2. Create PVC with 'accessModes: ReadWriteOnce'
+        3. Create two pods using same PVC
```

Comment:
Make it explicit to create these pods on two different nodes, if that is the requirement.

Reply:
Updated the docstring